pyproject.toml

```
@@ -0,0 +1,63 @@
+[project]
+name = "rabbitpy"
+version = "2.0.1"
+description = "A pure python, thread-safe, minimalistic and pythonic RabbitMQ client library"
+readme = "README.rst"
+requires-python = ">=3.8"
+license = { file = "LICENSE" }
+authors = [ { name = "Gavin M. Roy", email = "gavinmroy@gmail.com" } ]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Programming Language :: Python :: Implementation :: PyPy",
+    "Topic :: Communications",
+    "Topic :: Internet",
+    "Topic :: Software Development :: Libraries"
+]
+dependencies = [
+  "pamqp>=2.3.0,<3.0",
+]
+
+[project.optional-dependencies]
+dev = [
+    "codecov",
+    "coverage",
+    "mock",
+    "flake8",
+    "flake8-comprehensions",
+    "flake8-deprecated",
+    "flake8-html",
+    "flake8-import-order",
```

Similar support so that flake8 reads from _pyproject.toml_
```suggestion
    "flake8-import-order",
    "flake8-pyproject",
```

---

Addressed in f7aab55